iron-dropdown-scroll-manager.html

```diff
@@ -172,7 +172,7 @@
       _scrollInteractionHandler: function(event) {
         if (Polymer
               .IronDropdownScrollManager
-              .elementIsScrollLocked(event.target)) {
+              .elementIsScrollLocked(event.path[0])) {
```

Comment:
yes, @cdata's PR makes more sense as I think using .path[] directly will throw in browsers that don't yet support it:
http://caniuse.com/#feat=shadowdom
http://w3c.github.io/webcomponents/spec/shadow/#event-paths